source/Outfit.cpp

```diff
@@ -372,6 +386,83 @@ const string &Outfit::Description() const
 
 
 
+uint32_t Outfit::PassengerBunks() const
+{
+	uint32_t passengers = bunks;
+	for(auto bunkType : bunkTypes)
+	{
+		if(GameData::BunkType(bunkType.first).second.first)
+			passengers += bunkType.second;
+	}
+	return passengers;
+}
+
+
+
+uint32_t Outfit::CrewBunks() const
+{
+	uint32_t crew = bunks;
+	for(auto bunkType : bunkTypes)
+	{
+		if(GameData::BunkType(bunkType.first).second.second)
+			crew += bunkType.second;
+	}
```

Comment:
```suggestion
		if(GameData::BunkType(bunkType.first).second.second)
			crew += bunkType.second;
```
You dont actually need these braces, as the if is contained within the for (and considered as a single line command)